modules/RTC/TraceablePeerConnection.js

```diff
@@ -572,6 +569,33 @@ TraceablePeerConnection.prototype._peerMutedChanged = function(
     }
 };
 
+/**
+ * Handles remote source mute, unmute and videoType changed events.
+ *
+ * @param {string} sourceName - The name of the remote source.
+ * @param {boolean} isMuted - The new mute state.
+ * @param {string} videoType - The new videoType value.
+ */
+TraceablePeerConnection.prototype._sourceMutedOrVideoTypeChanged = function(sourceName, isMuted, videoType) {
```

Comment:
can you split this into 2: source muted and source video type changed methods?

Reply:
I had it that way before but its code repetition and there is only 1 line that is different.

Reply:
The code repetition is only for the track lookup which calls for a utility method eventually.